src/Microsoft.DotNet.Build.Tasks/PackageFiles/publishtest.targets

```diff
@@ -12,12 +12,6 @@
     <TargetOS Condition="'$(OSGroup)'!='AnyOS'">$(OSGroup)</TargetOS>
   </PropertyGroup>
 
-  <Target Name="RestoreTestRuntimePackage"
```

Comment:
I can try allowing a fallback, but I'm not sure it will 100% work with all of the other changes here. I will see if I can get WCF to work without any changes in that repo. I was planning on having another fix for WCF ready shortly after this goes in. I don't think there will be very much churn there, because there are not very many test projects, compared to corefx.